File: src/libs/ThirdParty.VBSuppliers/Properties/ExpediaRapid/SerializableClasses/Search/Promotions.vb
Imports Newtonsoft.Json
Namespace ExpediaRapid.SerializableClasses.Search

    Public Class Promotions

        <JsonProperty("value_adds")>
        Public Property ValueAdds As New Dictionary(Of String, Promotion)

        <JsonProperty("deal")>
        Public Property Deal As Promotion

    End Class

End Namespace

File: src/libs/iVectorOne.Suppliers/ExpediaRapid/SerializableClasses/Search/Promotions.cs
namespace ThirdParty.CSSuppliers.ExpediaRapid.SerializableClasses.Search
{
    using System.Collections.Generic;
    using Newtonsoft.Json;

    public class Promotions
    {

        [JsonProperty("value_adds")]
        public Dictionary<string, Promotion> ValueAdds { get; set; } = new Dictionary<string, Promotion>();

        [JsonProperty("deal")]
        public Promotion Deal { get; set; }

    }

}